indico/modules/events/roles/util.py

```diff
@@ -25,3 +32,31 @@ def serialize_event_role(role, legacy=True):
             'name': role.name,
             'identifier': 'EventRole:{}'.format(role.id),
         }
+
+
+def import_members_from_csv(f):
+    """Import members from a CSV file into an event role."""
+    reader = csv.reader(f.read().splitlines())
+    members_data = set()
+
+    for num_row, row in enumerate(reader, 1):
+        try:
+            email = [to_unicode(value).strip() for value in row]
```

Comment:
You expect only one row, so simply use `email = to_unicode(row[0]).strip().lower()` directly (and possibly abort if `len(row) != 1`)